common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/Prisoner.kt

```diff
@@ -469,3 +469,6 @@ class PrisonerA : Prisoner()
 
 @Document(indexName = "prisoner-search-b")
 class PrisonerB : Prisoner()
+
+@Document(indexName = "prisoner-search")
+class PrisonerC : Prisoner()
```

Comment:
Not sure why the existing index names dont match the SyncIndex index names but this one does

Reply:
don't think these are used though - think `-a` and `-b` can be removed instead

Reply:
Great, have removed.